src/server/pachyderm_test.go

```diff
@@ -255,20 +255,22 @@ func TestPipelineWithParallelism(t *testing.T) {
 	require.NoError(t, c.FinishCommit(dataRepo, commit1.ID))
 
 	pipeline := tu.UniqueString("pipeline")
-	require.NoError(t, c.CreatePipeline(
-		pipeline,
-		"",
-		[]string{"bash"},
-		[]string{
-			fmt.Sprintf("cp /pfs/%s/* /pfs/out/", dataRepo),
-		},
-		&pps.ParallelismSpec{
-			Constant: 4,
+	_, err = c.PpsAPIClient.CreatePipeline(context.Background(),
+		&pps.CreatePipelineRequest{
```

Comment:
Oh I changed this to set `standby: true` because that was how I was testing it for a while. I'll change it back because I'm writing a whole new test for this and there's no reason to change this one.

Reply:
Changed back now.